ops/ansible/playbooks-ccs/build_bfd-pipeline.yml

```diff
@@ -9,6 +9,7 @@
   gather_facts: yes
   vars:
     ansible_ssh_pipelining: no
+    bfd_pipeline_dir: '/usr/local/bfd-pipeline'
```

Comment:
Are we moving the pipeline's install directory from `/bluebutton-data-pipeline` to here?

Reply:
TODO update splunk and cloudwatch